### PR TITLE
integrating go-commons prometheus

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Integrating go-commons prometheus client
Prerequisite PR: https://github.com/cloud-bulldozer/go-commons/pull/8
We will change the import once the prerequisite PR is merged and cut a new release. Thanks!

### Tested in local environment
```
[vchalla@vchalla kube-burner]$ kube-burner index -c ~/kube-burner-config.yaml --job-name=node-density-heavy -e ~/metrics-endpoints.yaml
time="2023-05-19 15:53:27" level=info msg="📁 Creating indexer: elastic" file="metrics.go:39"
time="2023-05-19 15:53:28" level=info msg="👽 Initializing prometheus client with URL: https://prometheus-k8s-openshift-monitoring.apps.vchalla-quay-test-3.perfscale.devcluster.openshift.com" file="metrics.go:77"
time="2023-05-19 15:53:28" level=info msg="Scraping for the prometheus entry with params - Endpoint: https://prometheus-k8s-openshift-monitoring.apps.vchalla-quay-test-3.perfscale.devcluster.openshift.com, Profile: /home/vchalla/e2e-benchmarking/workloads/kube-burner/metrics-profiles/metrics.yaml, Start: 2023-05-19T14:53:27-04:00, End: 2023-05-19T15:53:27-04:00" file="utils.go:82"
time="2023-05-19 15:53:34" level=info msg="Indexing metrics with UUID d27feb77-3f60-4ba8-b6dd-2e34dab39672" file="utils.go:88"
time="2023-05-19 15:53:34" level=info msg="Indexing metric schedulingThroughput" file="utils.go:93"
time="2023-05-19 15:53:35" level=info msg="Indexing finished in 1.019s: created=22" file="utils.go:99"
time="2023-05-19 15:53:35" level=info msg="Indexing metric readOnlyAPICallsLatency" file="utils.go:93"
time="2023-05-19 15:53:53" level=info msg="Indexing finished in 18.16s: created=8387" file="utils.go:99"
time="2023-05-19 15:53:53" level=info msg="Indexing metric mutatingAPICallsLatency" file="utils.go:93"
time="2023-05-19 15:53:59" level=info msg="Indexing finished in 5.675s: created=3733" file="utils.go:99"
time="2023-05-19 15:53:59" level=info msg="Indexing metric APIRequestRate" file="utils.go:93"
time="2023-05-19 15:54:13" level=info msg="Indexing finished in 14.047s: created=14287" file="utils.go:99"
time="2023-05-19 15:54:13" level=info msg="Indexing metric serviceSyncLatency" file="utils.go:93"
time="2023-05-19 15:54:13" level=info msg="Indexing finished in 0s:" file="utils.go:99"
time="2023-05-19 15:54:13" level=info msg="Indexing metric containerCPU" file="utils.go:93"
time="2023-05-19 15:54:23" level=info msg="Indexing finished in 10.238s: created=13054" file="utils.go:99"
time="2023-05-19 15:54:23" level=info msg="Indexing metric containerMemory" file="utils.go:93"
time="2023-05-19 15:54:33" level=info msg="Indexing finished in 9.692s: created=15125" file="utils.go:99"
time="2023-05-19 15:54:33" level=info msg="Indexing metric kubeletCPU" file="utils.go:93"
time="2023-05-19 15:54:34" level=info msg="Indexing finished in 1.314s: created=363" file="utils.go:99"
time="2023-05-19 15:54:34" level=info msg="Indexing metric kubeletMemory" file="utils.go:93"
time="2023-05-19 15:54:35" level=info msg="Indexing finished in 1.191s: created=363" file="utils.go:99"
time="2023-05-19 15:54:35" level=info msg="Indexing metric crioCPU" file="utils.go:93"
time="2023-05-19 15:54:37" level=info msg="Indexing finished in 1.204s: created=363" file="utils.go:99"
time="2023-05-19 15:54:37" level=info msg="Indexing metric crioMemory" file="utils.go:93"
time="2023-05-19 15:54:38" level=info msg="Indexing finished in 1.192s: created=363" file="utils.go:99"
time="2023-05-19 15:54:38" level=info msg="Indexing metric containerNetworkSetupLatency" file="utils.go:93"
time="2023-05-19 15:54:39" level=info msg="Indexing finished in 1.001s: created=26" file="utils.go:99"
time="2023-05-19 15:54:39" level=info msg="Indexing metric nodeCPU-Workers" file="utils.go:93"
time="2023-05-19 15:54:41" level=info msg="Indexing finished in 2.176s: created=2357" file="utils.go:99"
time="2023-05-19 15:54:41" level=info msg="Indexing metric nodeCPU-Masters" file="utils.go:93"
time="2023-05-19 15:54:43" level=info msg="Indexing finished in 2.195s: created=2449" file="utils.go:99"
time="2023-05-19 15:54:43" level=info msg="Indexing metric nodeCPU-Infra" file="utils.go:93"
time="2023-05-19 15:54:43" level=info msg="Indexing finished in 0s:" file="utils.go:99"
time="2023-05-19 15:54:43" level=info msg="Indexing metric nodeMemoryAvailable-Masters" file="utils.go:93"
time="2023-05-19 15:54:45" level=info msg="Indexing finished in 1.222s: created=363" file="utils.go:99"
time="2023-05-19 15:54:45" level=info msg="Indexing metric nodeMemoryAvailable-Workers" file="utils.go:93"
time="2023-05-19 15:54:46" level=info msg="Indexing finished in 1.203s: created=363" file="utils.go:99"
time="2023-05-19 15:54:46" level=info msg="Indexing metric nodeMemoryUtilization-Workers" file="utils.go:93"
time="2023-05-19 15:54:47" level=info msg="Indexing finished in 1.231s: created=363" file="utils.go:99"
time="2023-05-19 15:54:47" level=info msg="Indexing metric nodeMemoryAvailable-Infra" file="utils.go:93"
time="2023-05-19 15:54:47" level=info msg="Indexing finished in 0s:" file="utils.go:99"
time="2023-05-19 15:54:47" level=info msg="Indexing metric nodeMemoryTotal-Masters" file="utils.go:93"
time="2023-05-19 15:54:48" level=info msg="Indexing finished in 1.213s: created=363" file="utils.go:99"
time="2023-05-19 15:54:48" level=info msg="Indexing metric nodeMemoryTotal-Workers" file="utils.go:93"
time="2023-05-19 15:54:49" level=info msg="Indexing finished in 1.223s: created=363" file="utils.go:99"
time="2023-05-19 15:54:49" level=info msg="Indexing metric nodeMemoryTotal-Infra" file="utils.go:93"
time="2023-05-19 15:54:49" level=info msg="Indexing finished in 0s:" file="utils.go:99"
time="2023-05-19 15:54:49" level=info msg="Indexing metric etcdLeaderChangesRate" file="utils.go:93"
time="2023-05-19 15:54:51" level=info msg="Indexing finished in 1.074s: created=121" file="utils.go:99"
time="2023-05-19 15:54:51" level=info msg="Indexing metric 99thEtcdDiskBackendCommitDurationSeconds" file="utils.go:93"
time="2023-05-19 15:54:52" level=info msg="Indexing finished in 1.243s: created=363" file="utils.go:99"
time="2023-05-19 15:54:52" level=info msg="Indexing metric 99thEtcdDiskWalFsyncDurationSeconds" file="utils.go:93"
time="2023-05-19 15:54:53" level=info msg="Indexing finished in 1.211s: created=363" file="utils.go:99"
time="2023-05-19 15:54:53" level=info msg="Indexing metric 99thEtcdRoundTripTimeSeconds" file="utils.go:93"
time="2023-05-19 15:54:54" level=info msg="Indexing finished in 1.44s: created=726" file="utils.go:99"
time="2023-05-19 15:54:54" level=info msg="Indexing metric etcdVersion" file="utils.go:93"
time="2023-05-19 15:54:55" level=info msg="Indexing finished in 83ms: created=1" file="utils.go:99"
time="2023-05-19 15:54:55" level=info msg="Indexing metric namespaceCount" file="utils.go:93"
time="2023-05-19 15:54:56" level=info msg="Indexing finished in 1.133s: created=121" file="utils.go:99"
time="2023-05-19 15:54:56" level=info msg="Indexing metric podStatusCount" file="utils.go:93"
time="2023-05-19 15:54:57" level=info msg="Indexing finished in 1.471s: created=605" file="utils.go:99"
time="2023-05-19 15:54:57" level=info msg="Indexing metric secretCount" file="utils.go:93"
time="2023-05-19 15:54:57" level=info msg="Indexing finished in 81ms: created=1" file="utils.go:99"
time="2023-05-19 15:54:57" level=info msg="Indexing metric deploymentCount" file="utils.go:93"
time="2023-05-19 15:54:57" level=info msg="Indexing finished in 82ms: created=1" file="utils.go:99"
time="2023-05-19 15:54:57" level=info msg="Indexing metric configmapCount" file="utils.go:93"
time="2023-05-19 15:54:57" level=info msg="Indexing finished in 81ms: created=1" file="utils.go:99"
time="2023-05-19 15:54:57" level=info msg="Indexing metric serviceCount" file="utils.go:93"
time="2023-05-19 15:54:58" level=info msg="Indexing finished in 82ms: created=1" file="utils.go:99"
time="2023-05-19 15:54:58" level=info msg="Indexing metric nodeRoles" file="utils.go:93"
time="2023-05-19 15:54:59" level=info msg="Indexing finished in 1.644s: created=1089" file="utils.go:99"
time="2023-05-19 15:54:59" level=info msg="Indexing metric nodeStatus" file="utils.go:93"
time="2023-05-19 15:55:01" level=info msg="Indexing finished in 1.289s: created=484" file="utils.go:99"
time="2023-05-19 15:55:01" level=info msg="Indexing metric podDistribution" file="utils.go:93"
time="2023-05-19 15:55:02" level=info msg="Indexing finished in 1.622s: created=726" file="utils.go:99"
time="2023-05-19 15:55:02" level=info msg="Indexing metric clusterVersion" file="utils.go:93"
time="2023-05-19 15:55:02" level=info msg="Indexing finished in 84ms: created=1" file="utils.go:99"
time="2023-05-19 15:55:02" level=info msg="👋 Exiting kube-burner d27feb77-3f60-4ba8-b6dd-2e34dab39672" file="kube-burner.go:198"

```

### CODE REVIEW

File: pkg/prometheus/prometheus.go [hunk 1]
```diff
@@ -155,109 +106,86 @@ func (p *Prometheus) ScrapeJobsMetrics(indexer *indexers.Indexer) error {
...
 	if !ok {
 		return fmt.Errorf("unsupported result format: %s", value.Type().String())
 	}
-	for _, v := range data {
-		for _, prometheusJob := range p.JobList {
-			if v.Timestamp.Time().Before(prometheusJob.End) {
-				jobName = prometheusJob.Name
-				jobConfig = prometheusJob.JobConfig
-				jobConfig.NamespaceLabels = nil // no need to insert this into the metric
-				jobConfig.Objects = nil         // no need to insert this into the metric
```
Comment (rsevilla87) [inline pkg/prometheus/prometheus.go:171]: I see no reference to these 2 lines in createMetric()

**Review by rsevilla87 — CHANGES_REQUESTED** [review]
Hi!, I added some comments

File: pkg/prometheus/prometheus.go (hunk 1 above)
Comment (vishnuchalla) [reply pkg/prometheus/prometheus.go:171]: Yes. As mentioned above, by default their value will be nil, so did not set them.

File: pkg/prometheus/prometheus.go [hunk 2]
```diff
@@ -155,109 +106,86 @@ func (p *Prometheus) ScrapeJobsMetrics(indexer *indexers.Indexer) error {
...
-	if err != nil {
-		return v, err
+// Create metric creates metric to be indexed
+func createMetric(uuid, query, metricName, jobName string, jobConfig config.Job, metadata map[string]interface{}, labels model.Metric, value model.SampleValue, timestamp time.Time) metric {
+	m := metric{
+		Labels:     make(map[string]string),
+		UUID:       uuid,
+		Query:      query,
+		MetricName: metricName,
+		JobName:    jobName,
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply pkg/prometheus/prometheus.go]: Changed as suggested.

File: pkg/util/metrics/utils.go [hunk 3]
```diff
@@ -19,11 +19,11 @@ import (
 	"io"
 	"time"
 
-	"github.com/cloud-bulldozer/go-commons/indexers"
 	"github.com/cloud-bulldozer/kube-burner/pkg/config"
 	"github.com/cloud-bulldozer/kube-burner/pkg/prometheus"
 	"github.com/cloud-bulldozer/kube-burner/pkg/util"
 	log "github.com/sirupsen/logrus"
+	"github.com/vishnuchalla/go-commons/indexers"
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply pkg/util/metrics/utils.go]: Its just go formatting. Will be fixed when used the right import.

File: pkg/prometheus/prometheus.go [hunk 4]
```diff
@@ -155,109 +106,86 @@ func (p *Prometheus) ScrapeJobsMetrics(indexer *indexers.Indexer) error {
...
 		return fmt.Errorf("unsupported result format: %s", value.Type().String())
 	}
-	for _, v := range data {
-		for _, val := range v.Values {
-			for _, job := range p.JobList {
-				if val.Timestamp.Time().Before(job.End) {
-					jobName = job.Name
-					jobConfig = job.JobConfig
-					jobConfig.NamespaceLabels = nil // no need to insert this into the metric
-					jobConfig.Objects = nil         // no need to insert this into the metric.
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply pkg/prometheus/prometheus.go:214]: Yes. By default their value will be nil, so did not set them.

File: pkg/prometheus/prometheus.go [hunk 5]
```diff
@@ -155,109 +106,86 @@ func (p *Prometheus) ScrapeJobsMetrics(indexer *indexers.Indexer) error {
...
 
-// QueryRange prometheus queryRange wrapper
-func (p *Prometheus) QueryRange(query string, start, end time.Time) (model.Value, error) {
-	var v model.Value
-	r := apiv1.Range{Start: start, End: end, Step: p.Step}
-	v, _, err := p.api.QueryRange(context.TODO(), query, r)
-	if err != nil {
-		return v, err
+// Create metric creates metric to be indexed
+func createMetric(uuid, query, metricName, jobName string, jobConfig config.Job, metadata map[string]interface{}, labels model.Metric, value model.SampleValue, timestamp time.Time) metric {
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply pkg/prometheus/prometheus.go]: Changed as suggested, which reduced some amount of arguments.

**Comment by rsevilla87** [comment]
> Regarding the queries on `ClusterMetadata`, had to make that change to fix golangci-lint errors as below. Please feel free to let me know if you have any other thoughts on this. Thanks!
> 
> ```
> pkg/workloads/helpers.go:41:2: undefined: ocpmetadata (typecheck)
> 	ocpmetadata.ClusterMetadata
> 	^
> pkg/workloads/helpers.go:133:27: wh.Metadata.MetricName undefined (type BenchmarkMetadata has no field or method MetricName) (typecheck)
> 		MetricName: wh.Metadata.MetricName,
> 		                        ^
> pkg/workloads/helpers.go:145:29: wh.Metadata.Platform undefined (type BenchmarkMetadata has no field or method Platform) (typecheck)
> 		"platform":   wh.Metadata.Platform,
> 		                          ^
> pkg/workloads/helpers.go:146:29: wh.Metadata.OCPVersion undefined (type BenchmarkMetadata has no field or method OCPVersion) (typecheck)
> 		"ocpVersion": wh.Metadata.OCPVersion,
> 		                          ^
> pkg/workloads/helpers.go:147:29: wh.Metadata.K8SVersion undefined (type BenchmarkMetadata has no field or method K8SVersion) (typecheck)
> 		"k8sVersion": wh.Metadata.K8SVersion,
> 		                          ^
> pkg/workloads/helpers.go:148:29: wh.Metadata.TotalNodes undefined (type BenchmarkMetadata has no field or method TotalNodes) (typecheck)
> 		"totalNodes": wh.Metadata.TotalNodes,
> 		                          ^
> pkg/workloads/helpers.go:149:29: wh.Metadata.SDNType undefined (type BenchmarkMetadata has no field or method SDNType) (typecheck)
> 		"sdnType":    wh.Metadata.SDNType,
> ```

Somethis is not working well, I don't see any indexed document in ElasticSearch from the OCP wrapper tests.

File: pkg/workloads/helpers.go [hunk 6]
```diff
@@ -40,13 +40,13 @@ const (
...
 
 type BenchmarkMetadata struct {
-	ocpmetadata.ClusterMetadata
-	UUID         string                 `json:"uuid"`
-	Benchmark    string                 `json:"benchmark"`
-	Timestamp    time.Time              `json:"timestamp"`
-	EndDate      time.Time              `json:"endDate"`
-	Passed       bool                   `json:"passed"`
-	UserMetadata map[string]interface{} `json:"metadata,omitempty"`
+	ClusterMetadata ocpmetadata.ClusterMetadata
```
[the remark this answers is not in the document]
Comment (rsevilla87) [reply pkg/workloads/helpers.go:43]: It was passing linters before this PR

**Comment by vishnuchalla** [comment]
> > Regarding the queries on `ClusterMetadata`, had to make that change to fix golangci-lint errors as below. Please feel free to let me know if you have any other thoughts on this. Thanks!
> > ```
> > pkg/workloads/helpers.go:41:2: undefined: ocpmetadata (typecheck)
> > 	ocpmetadata.ClusterMetadata
> > 	^
> > pkg/workloads/helpers.go:133:27: wh.Metadata.MetricName undefined (type BenchmarkMetadata has no field or method MetricName) (typecheck)
> > 		MetricName: wh.Metadata.MetricName,
> > 		                        ^
> > pkg/workloads/helpers.go:145:29: wh.Metadata.Platform undefined (type BenchmarkMetadata has no field or method Platform) (typecheck)
> > 		"platform":   wh.Metadata.Platform,
> > 		                          ^
> > pkg/workloads/helpers.go:146:29: wh.Metadata.OCPVersion undefined (type BenchmarkMetadata has no field or method OCPVersion) (typecheck)
> > 		"ocpVersion": wh.Metadata.OCPVersion,
> > 		                          ^
> > pkg/workloads/helpers.go:147:29: wh.Metadata.K8SVersion undefined (type BenchmarkMetadata has no field or method K8SVersion) (typecheck)
> > 		"k8sVersion": wh.Metadata.K8SVersion,
> > 		                          ^
> > pkg/workloads/helpers.go:148:29: wh.Metadata.TotalNodes undefined (type BenchmarkMetadata has no field or method TotalNodes) (typecheck)
> > 		"totalNodes": wh.Metadata.TotalNodes,
> > 		                          ^
> > pkg/workloads/helpers.go:149:29: wh.Metadata.SDNType undefined (type BenchmarkMetadata has no field or method SDNType) (typecheck)
> > 		"sdnType":    wh.Metadata.SDNType,
> > ```
> 
> Something is not working well, I don't see any indexed document in ElasticSearch from the OCP wrapper tests.

@rsevilla87  It was not realted to the changes in this PR. The root cause for no indexing in ocp tests was an incorrect declaration and usage of a [variable](https://github.com/cloud-bulldozer/kube-burner/blob/master/test/run-ocp.sh#L19) in our test script, which was introduced in one of our old [PRs](https://github.com/cloud-bulldozer/kube-burner/pull/318) as a part of correcting lint errors. Now with the new fix in the script we are able to see the index functionality working. And also I feel like having `ClusterMetadata` a name will make it more verbose and help the devs understand on where the variable is coming from rather than embedding it. But, happy to keep it embedded if we still want it. Thanks!

File: pkg/prometheus/prometheus.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (rsevilla87) [reply pkg/prometheus/prometheus.go:214]: Their value is not nil, I've seen documents indexed with these fields and we wanted to remove them.

**Comment by rsevilla87** [comment]
> lso I feel like having `ClusterMetadata` a name will make it more verbose and help the devs understand on where the varia

The field ClusterMetadata must be embed, assigning a field name to it would generate a different document and making it not backwards compatible. e.g.

This is what's being generated now with this PR:

```json
[
  {
    "ClusterMetadata": {
      "metricName": "clusterMetadata",
      "platform": "AWS",
      "ocpVersion": "4.11.17",
      "k8sVersion": "v1.24.6+5658434",
      "masterNodesType": "m5.2xlarge",
      "workerNodesType": "m5.2xlarge",
      "masterNodesCount": 3,
      "infraNodesType": "",
      "workerNodesCount": 3,
      "infraNodesCount": 0,
      "totalNodes": 3,
      "sdnType": "OpenShiftSDN",
      "clusterName": "cloud-bulldozer-ci-6hwqr",
      "region": "us-west-2"
    },
    "uuid": "4a23d767-858a-45b2-9e45-bd545f75a648",
    "benchmark": "cluster-density",
    "timestamp": "2023-06-02T09:16:55.070561012Z",
    "endDate": "2023-06-02T09:18:17.244139437Z",
    "passed": true
  }
]
```


And this is what it was being generated originally:

```json
{
    "metricName": "clusterMetadata",
    "platform": "AWS",
    "ocpVersion": "4.11.17",
    "k8sVersion": "v1.24.6+5658434",
    "masterNodesType": "m5.2xlarge",
    "workerNodesType": "m5.2xlarge",
    "masterNodesCount": 3,
    "infraNodesType": "",
    "workerNodesCount": 3,
    "infraNodesCount": 0,
    "totalNodes": 3,
    "sdnType": "OpenShiftSDN",
    "clusterName": "cloud-bulldozer-ci-6hwqr",
    "region": "us-west-2",
    "uuid": "4a75b932-bfc3-4e7c-b63d-43ad37723bd9",
    "benchmark": "cluster-density",
    "timestamp": "2023-05-17T13:45:06.663792925Z",
    "endDate": "2023-05-17T13:45:08.688147752Z",
    "passed": false
  }
```

File: pkg/prometheus/prometheus.go (hunk 4 above)
[the remark this answers is not in the document]
Comment (rsevilla87) [reply pkg/prometheus/prometheus.go:214]:
```json
$ cat collected-metrics/etcdVersion.json  | jq .
[                        
  {                          
    "timestamp": "2023-06-02T11:17:59.396+02:00",
    "labels": {            
      "cluster_version": "3.5"
    },   
    "value": 3,                             
    "uuid": "4a23d767-858a-45b2-9e45-bd545f75a648",
    "query": "sum by (cluster_version)(etcd_cluster_version)",
    "metricName": "etcdVersion",
    "jobName": "cluster-density",
    "jobConfig": {
      "jobIterations": 2,
      "name": "cluster-density",
      "objects": [
        {         
          "objectTemplate": "imagestream.yml",
          "replicas": 1,               
          "namespaced": true, 
          "wait": true,        
          "waitOptions": {}
        },                         
        {                         
etc.
```

we don't want to index arbitrary fields like objects and namespaceLabels

[the remark this answers is not in the document]
Comment (vishnuchalla) [reply pkg/prometheus/prometheus.go:214]: Understood the context and changed as suggested. Thanks!

**Comment by vishnuchalla** [comment]
> > lso I feel like having `ClusterMetadata` a name will make it more verbose and help the devs understand on where the varia
> 
> The field ClusterMetadata must be embed, assigning a field name to it would generate a different document and making it not backwards compatible. e.g.
> 
> This is what's being generated now with this PR:
> 
> ```json
> [
>   {
>     "ClusterMetadata": {
>       "metricName": "clusterMetadata",
>       "platform": "AWS",
>       "ocpVersion": "4.11.17",
>       "k8sVersion": "v1.24.6+5658434",
>       "masterNodesType": "m5.2xlarge",
>       "workerNodesType": "m5.2xlarge",
>       "masterNodesCount": 3,
>       "infraNodesType": "",
>       "workerNodesCount": 3,
>       "infraNodesCount": 0,
>       "totalNodes": 3,
>       "sdnType": "OpenShiftSDN",
>       "clusterName": "cloud-bulldozer-ci-6hwqr",
>       "region": "us-west-2"
>     },
>     "uuid": "4a23d767-858a-45b2-9e45-bd545f75a648",
>     "benchmark": "cluster-density",
>     "timestamp": "2023-06-02T09:16:55.070561012Z",
>     "endDate": "2023-06-02T09:18:17.244139437Z",
>     "passed": true
>   }
> ]
> ```
> 
> And this is what it was being generated originally:
> 
> ```json
> {
>     "metricName": "clusterMetadata",
>     "platform": "AWS",
>     "ocpVersion": "4.11.17",
>     "k8sVersion": "v1.24.6+5658434",
>     "masterNodesType": "m5.2xlarge",
>     "workerNodesType": "m5.2xlarge",
>     "masterNodesCount": 3,
>     "infraNodesType": "",
>     "workerNodesCount": 3,
>     "infraNodesCount": 0,
>     "totalNodes": 3,
>     "sdnType": "OpenShiftSDN",
>     "clusterName": "cloud-bulldozer-ci-6hwqr",
>     "region": "us-west-2",
>     "uuid": "4a75b932-bfc3-4e7c-b63d-43ad37723bd9",
>     "benchmark": "cluster-density",
>     "timestamp": "2023-05-17T13:45:06.663792925Z",
>     "endDate": "2023-05-17T13:45:08.688147752Z",
>     "passed": false
>   }
> ```

Changed as suggested.

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**